scripts/qa_crowbarsetup.sh

```diff
@@ -404,22 +404,16 @@ function export_tftpboot_repos_dir()
 
 function addsp3testupdates()
 {
-    add_mount "SLES11-SP3-Updates" 'you.suse.de:/you/http/download/x86_64/update/SLE-SERVER/11-SP3/' "$tftpboot_repos_dir/SLES11-SP3-Updates/" "sp3tup"
-}
-
-function addsp4testupdates()
-{
-    add_mount "SLES11-SP4-Updates" 'you.suse.de:/you/http/download/x86_64/update/SLE-SERVER/11-SP4/' "$tftpboot_repos_dir/SLES11-SP4-Updates/" "sp4tup"
+    add_mount "SLES11-SP3-Updates-test" \
+        'dist.suse.de:/dist/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/11-SP3:/x86_64/update/' \
+        "$tftpboot_repos_dir/SLES11-SP3-Updates-test/" "sp3tup"
 }
 
 function addsles12testupdates()
 {
-    echo "TODO: add SLES-12-GA Updates-test repo"
-}
-
-function addsles12sp1testupdates()
-{
```

Comment:
sle12sp1 test updates doesn't exist yet, and we're still sle12 ga based. I think its easier to just change s,ga,sp1, later..